pkg/kubelet/dockertools/manager_test.go

```diff
@@ -38,7 +38,7 @@ import (
 	"github.com/GoogleCloudPlatform/kubernetes/pkg/types"
 	"github.com/GoogleCloudPlatform/kubernetes/pkg/util"
 	uexec "github.com/GoogleCloudPlatform/kubernetes/pkg/util/exec"
-	"github.com/fsouza/go-dockerclient"
+	docker "github.com/fsouza/go-dockerclient"
```

Comment:
I see files with and without "docker". They seem to co-exist in the codebase. Do we want to converge on adding "docker"?